its/fixtures/pipeline-sonarcloud-v1-cli-6.2.1.4610-windows.yml

```diff
@@ -2,23 +2,24 @@
 # Config: {"os":"windows","version":{"extension":"sonarcloud","version":1},"scanner":{"type":"cli","version":"6.2.1.4610"}}
 
 trigger: none
+pr: none
 pool:
   vmImage: windows-latest
 variables:
   system.debug: true
 steps:
-  - task: SonarcloudPrepare@1
+  - task: SonarCloudPrepareTest@1
     inputs:
       SonarCloud: SonarCloud
-      organization: sonarsource-integration-tests
+      organization: sonarsource-sonar-scanner-azdo-it
```

Comment:
where is this org? 
https://dev.azure.com/sonarsource-sonar-scanner-azdo-it/ give me a 404 (do I need permissions)

Reply:
This is [this org](https://sonarcloud.io/organizations/sonarsource-sonar-scanner-azdo-it/projects?sort=-analysis_date) which you should have access to as owner